docs/csharp/language-reference/compiler-options/errors-warnings.md

```diff
@@ -44,7 +44,7 @@ The element value is the warning level you want displayed for the compilation: L
 | 1 | Displays severe warning messages. |
 | 2 | Displays level 1 warnings plus certain, less-severe warnings, such as warnings about hiding class members. |
 | 3 | Displays level 2 warnings plus certain, less-severe warnings, such as warnings about expressions that always evaluate to `true` or `false`. |
-| 4   (the default)|Displays all level 3 warnings plus informational warnings. |
+| 4 (default) | Displays all level 3 warnings plus informational warnings. |
```

Comment:
This formatting now matches the section below:

https://github.com/dotnet/docs/blob/89fd966130bd773bcbb97facc7b5e3ff4eb378b1/docs/csharp/language-reference/compiler-options/errors-warnings.md?plain=1#L61